index.html

```diff
@@ -1,227 +1,293 @@
+            <button><a href="">See project</a></button>
+          </div>
+        </div>
+      </div>
+      <div class="grid g9"></div>
+      <div class="grid g10"></div>
+      <div class="grid g2" id="inverse">
+        <div class="box2">
+          <h1>Uber Navigation</h1>
+          <div class="uber">
+            <p class="p1">UBER</p>
+            <p class="p2">Lead Developer</p>
+            <p class="p3">2018</p>
+          </div>
+          <h2>
+            A daily selection of privately personalized reads; no accounts or
+            sign-ups required.
+          </h2>
+          <div class="buttons">
+            <ul>
+              <li><button>html</button></li>
+              <li><button>css</button></li>
+              <li><button>javascript</button></li>
+            </ul>
+          </div>
+          <div class="project">
+            <button><a href="">See project</a></button>
+          </div>
+        </div>
+        <div class="box1">
+          <img class="image1" src="/images/imagen4.png" />
+        </div>
+      </div>
+      <div class="grid g12"></div>
+    </div>
+    <div class="about" id="about">
+      <div class="about1">
+        <p class="am">About Myself</p>
+        <p class="intro2">
+          Hello I'm a software developer! I can help you build a product ,
+          feature or website Look through some of my work and experience! If you
+          like what you see and have a project you need coded, don't hesitate to
+          contact me.
+        </p>
+        <p class="lc">Lets Connect</p>
+        <div class="icons2">
+          <div class="icon1"><i class="fa-brands fa-twitter"></i></div>
+          <div class="icon2"><i class="fa-brands fa-linkedin-in"></i></div>
+          <div class="icon5"><i class="fa-regular fa-hand-peace"></i></div>
+          <div class="icon3"><i class="fa-regular fa-m"></i></div>
+          <div class="icon4"><i class="fa-brands fa-github"></i></div>
+        </div>
+        <button class="resume">Get my resume</button>
+      </div>
+      <div class="about2">
+        <div class="languajes">
+          <button class="boton"><p>Languages</p></button>
+          <div class="arrows"><p>></p></div>
+          <div class="dropmenu1">
+            <ul>
+              <li>
+                <div class="java">
+                  <i class="fa-brands fa-square-js"></i>
+                  <p>JavaScript</p>
                 </div>
-                <h2>
-                    A daily selection of privately personalized reads; no accounts or sign-ups required.
-                </h2>
-                <div class="buttons">
-                    <ul>
-                        <li><button>html</button></li>
-                        <li><button>css</button></li>
-                        <li><button>javascript</button></li>
-                    </ul>
+              </li>
+              <li>
+                <div class="html">
+                  <i class="fa-brands fa-html5"></i>
```

Comment:
- [ ] Great job with your creativity. it is required that the list of languages under the language heading should display at page first page load, not when you hover the mouse ovet the language heading. Kindly make them to display when the page loads.
**NB:** _If you prefer it that way, you can reverse it once the project get approval_.

### Your Design
![your-lang](https://user-images.githubusercontent.com/26597432/228666298-e3d12f09-6d0f-48bc-a01b-b977b11eb515.png)

### Figma Design
![temp-lang](https://user-images.githubusercontent.com/26597432/228666324-05d18da0-d605-490f-8ae0-ba337a80db88.png)